docs/GettingStarted.md

```diff
@@ -179,4 +179,4 @@ module.exports = {
 };
 ```
 
-However, there are some [caveats](https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats) to using TypeScript with Babel. Because TypeScript support in Babel is transpilation, Jest will not type-check your tests as they are run. If you want that, you can use [ts-jest](https://github.com/kulshekhar/ts-jest) or setup a test runner using [jest-runner-tsc](https://github.com/azz/jest-runner-tsc).
+However, there are some [caveats](https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats) to using TypeScript with Babel. Because TypeScript support in Babel is purely transpilation, Jest will not type-check your tests as they are run. If you want that, you can use [ts-jest](https://github.com/kulshekhar/ts-jest) instead, or setup a separate type-checking runner using [jest-runner-tsc](https://github.com/azz/jest-runner-tsc).
```

Comment:
I'm don't think we should recommend the runner - it has serious drawbacks compared to using `tsc` directly, and #10704 is better served by `ts-jest`

Reply:
I guess this is a matter of opinion though? I would never use `ts-jest`, I view type-checking as a completely separate process that is more like linting or dynamic testing that I don't run all the time while I'm on WIP things, like I would with tests in TDD.

Reply:
I already tried to adjust the phrasing to make it easier for people to recognize the difference and click the link that will be more relevant to them

Reply:
What are the drawbacks though? I'm ready to be convinced if not the idea, but the concrete project `jest-runner-tsc` just doesn't work well. Never used it myself, only read the readme which doesn't say a lot.

Reply:
See https://github.com/azz/jest-runner-tsc/issues/20

Reply:
Okay, in that case, maybe it's just not `jest-runner-tsc`s time yet. Agree with closing this then, thanks for your contribution anyway @akphi even if we've decided not to include this link in the docs at this point :)